Sources/Web3Core/KeystoreManager/BIP32HDNode.swift

```diff
@@ -19,15 +19,15 @@ extension UInt32 {
         let byteArray = Array(bytePtr)
         return Data(byteArray)
     }
+    static var maxIterationIndex = UInt32(1) << 31
```

Comment:
Not the right place for this value as it has nothing to do with `UInt32`. Make it a static constant in `HDNode`.